### ISSUE TITLE
CS:GO Library Paths

### ISSUE THREAD

**Issue by sgtaziz** [devryan/GamePanelX-V3#106]
CS:GO released an update a few months back that somehow breaks Library Path pointers in srcds_run. This only happens when a folder is written as the ip (I'm guessing it doesn't handle the : too well.) Here is what I was able to find so far:

When the directory name is changed from IP to just a simple name (ie, test), it will boot.
In the new srcds_run file, the one that fails, this exists: `export LD_LIBRARY_PATH="${SRCDS_DIR}:${SRCDS_DIR}/bin:${LD_LIBRARY_PATH}"`
If you replace the above with this: `export LD_LIBRARY_PATH=".:bin:$LD_LIBRARY_PATH"` server boots up fine. 

In conclusion, I suggest setting up the Update script to change the LD_LIBRARY_PATH in CS:GO's srcds_run with the fixed path mentioned above. Thanks for the great project and I hope I helped!

**Comment by fabiango86** [comment]
How can i get a work around for this so i can make new users and adding cs go to them and it just starting fine?

**Comment by chandwer** [comment]
Currently experiencing this bug as well. What's the point of including the IP of the box in the path anyway? What happens of the server's IP changes?

**Comment by sgtaziz** [comment]
The file changes when the IP does. It's handy for quickly finding the server you want.
